kie-maven-takari-plugin/src/test/java/org/drools/compiler/xpath/tobeinstrumented/InjectReactiveMojoConfigTest.java

```diff
@@ -0,0 +1,130 @@
+package org.drools.compiler.xpath.tobeinstrumented;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.drools.core.phreak.ReactiveObject;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+import static org.kie.maven.plugin.InjectReactiveMojo.convertAllToPkgRegExps;
+import static org.kie.maven.plugin.InjectReactiveMojo.isPackageNameIncluded;
+
+public class InjectReactiveMojoConfigTest {
+
+    @Test
+    public void testRegexpForPackagesDefault() {
+        String[] inputConfig = new String[]{"*"};
+
+        List<String> config = convertAllToPkgRegExps(inputConfig);
+
+        System.out.println(Arrays.asList(config));
+
+        assertTrue(isPackageNameIncluded(Object.class.getPackage().getName(),
+                                         config));
+        assertTrue(isPackageNameIncluded(ReactiveObject.class.getPackage().getName(),
+                                         config));
+        assertTrue(isPackageNameIncluded("xyz.my",
+                                         config));
+    }
+
+    @Test
+    public void testRegexpForPackagesSingleNoStars() {
+        String[] inputConfig = new String[]{"org.drools"};
+
+        List<String> config = convertAllToPkgRegExps(inputConfig);
+
+        System.out.println(Arrays.asList(config));
```

Comment:
Please don't use `System.out.println` in tests. Either replace by logging or remove altogether.